skops/card/_markup.py

```diff
@@ -205,20 +231,34 @@ def _table_body(self, items) -> list[list[str]]:
             body.append(row)
         return body
 
-    def _table(self, item) -> str:
+    def _table_old(self, item) -> tuple[list[str], list[list[str]]]:  # pragma: no cover
+        # pandoc < 2.5
+        _, _, _, thead, tbody = item
+        columns = self._table_cols_old(thead)
+        body = self._table_body_old(tbody)
+        return columns, body
+
+    def _table_new(self, item) -> tuple[list[str], list[list[str]]]:
+        # pandoc >= 2.5
         # attr capt specs thead tbody tfoot
         _, _, _, thead, tbody, _ = item
-
         # header
         (_, thead_bodies) = thead
         (attr, thead_body) = thead_bodies[0]  # multiple headers?
-
-        columns = self._table_cols(thead_body)
-
+        columns = self._table_cols_new(thead_body)
```

Comment:
I would have expected these lines to be covered.

Reply:
So the table change actually happens after 2.9, not 2.5 as I first suspected. Therefore, given that we now install pandoc via apt on CI, which uses 2.9, the old table implementation is covered, not the new one. I thus moved around the pragmas to the new implementation.